robotiq_s_model_articulated_gazebo_plugins/src/RobotiqHandPlugin.cpp

```diff
@@ -0,0 +1,877 @@
+/*
+ * Copyright 2014 Open Source Robotics Foundation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+*/
+
+#include <robotiq_s_model_articulated_msgs/SModelRobotInput.h>
+#include <robotiq_s_model_articulated_msgs/SModelRobotOutput.h>
+#include <ros/ros.h>
+#include <string>
+#include <vector>
+#define private public 
```

Comment:
I cannot find this in [bitbucket.org/osrf/drcsim/src/drcsim_gazebo_ros_plugins/src/RobotiqHandPlugin.cpp](https://bitbucket.org/osrf/drcsim/src/7aee32718c4a74419b2bb447ab41d8ab9e4edf68/drcsim_gazebo_ros_plugins/src/RobotiqHandPlugin.cpp?at=default): could you explain why this is needed?